firetweet/src/main/java/org/getlantern/firetweet/util/CustomTabUtils.java

```diff
@@ -183,6 +186,12 @@ private static String getTagByType(@NonNull String type) {
             case TAB_TYPE_DIRECT_MESSAGES: {
```

Comment:
Yeah, I agree. I was just playing around with trying to get favorites/activity from friends to display with the existing implementation, for https://github.com/getlantern/firetweet/issues/113. I think part of it will need re-written. I went ahead and removed those changes from this commit anyway.